controllers/flowlogspipeline/flp_objects.go

```diff
@@ -259,6 +262,67 @@ func (b *builder) addTransformStages(lastStage *config.PipelineBuilderStage) {
 	})
 }
 
+func (b *builder) addMetricsStages(pipeline config.PipelineBuilderStage) {
```

Comment:
can we make changes to the `confGenerator` so that the output will fit easier here? for example, add a flag to `confGenerator` so that it is creating just a subset of the stages that fit the needs and we do not need to reread and parse the file???

Reply:
Actually, if this is not a lot of parsing, maybe we can keep this like that ... I want to get thoughts from @jotak , @jpinsonneau et al., on that part ... to me, it is better to get from `confGenerator` output that is very close to what we need but on the flip side, parsing a bit the output to fit into NOO is reasonable

Reply:
The current implementation simply extracts from the file produced by confGenerator and merges it with the rest of the current flp configuration.

Reply:
Changes made to confgenerator to provide only the required stages.